README.md

```diff
@@ -35,9 +35,23 @@ WAGTAILMARKDOWN_EXTENSIONS = ["toc", "sane_lists"]
 ### Installation
 Alpha release is available on Pypi - https://pypi.org/project/wagtail-markdown/ - installable via `pip install wagtail-markdown`. It's not a production ready release.
 
-The SimpleMDE editor uses FontAwesome for its widget icons. You can install [Wagtail FontAwesome](https://gitlab.com/alexgleason/wagtailfontawesome) via `pip install wagtailfontawesome`, or if you already have the stylesheet,  add the following to a `wagtail_hooks` module in a registered app in your project:
+The EasyMDE editor uses [FontAwesome 5](https://fontawesome.com/how-to-use/graphql-api/intro/getting-started) for its widget icons. You can get it with 
```

Comment:
According to

Locally it GETs:

https://maxcdn.bootstrapcdn.com/font-awesome/latest/css/font-awesome.min.css, then https://maxcdn.bootstrapcdn.com/font-awesome/latest/fonts/fontawesome-webfont.woff2?v=4.7.0

regardless whether you use wagtailfontawesome, or the manual download + hook as described below.

https://github.com/Ionaru/easy-markdown-editor/#options-list states:
> autoDownloadFontAwesome: If set to true, force downloads Font Awesome (used for icons). If set to false, prevents downloading. Defaults to undefined, which will intelligently check whether Font Awesome has already been included, then download accordingly.

So the best bet would be to initialize EasyMDE with `autoDownloadFontAwesome: false`